static/js/zpoilers.js

```diff
@@ -0,0 +1,62 @@
+function collapse_zpoiler(zpoiler) {
+    var zpoiler_height = zpoiler.prop('scrollHeight');
+
+    // Set height to rendered height on next frame, then to zero on following
+    // frame to allow CSS transition animation to work
+    requestAnimationFrame(function() {
+        zpoiler.height(zpoiler_height + 'px');
+        zpoiler.removeClass("zpoiler-content-open");
+        zpoiler.addClass("zpoiler-content-closed");
+
+        requestAnimationFrame(function() {
+            zpoiler.height("0px");
+        });
+    });
+}
+
+function expand_zpoiler(zpoiler) { 
+    var zpoiler_height = zpoiler.prop('scrollHeight');
+    zpoiler.height(zpoiler_height + "px");
+    zpoiler.removeClass("zpoiler-content-closed");
+    zpoiler.addClass("zpoiler-content-open");
```

Comment:
Maybe a brief comment belongs here that this is what triggers the animation for it to open?